build/packages/rpm/server/SPECS/victoriametrics.spec

```diff
@@ -53,7 +53,7 @@ install -D -p -m 0755 ./bin/vmalert-pure %{buildroot}%{_sbindir}/vmalert
 * Tue Dec 15 2020 Nurlan Moldomurov <nurlan.moldomurov@percona.com> - 1.93.1-1
 - upgrade victoriametrics to 1.93.1 release
 
-* Thu Mar 29 2023 Alexey Mukas <oleksii.mukas@ext.percona.com> - 1.89.1
+* Wed Mar 29 2023 Alexey Mukas <oleksii.mukas@ext.percona.com> - 1.89.1
```

Comment:
I wonder how you discovered this? 😄

Reply:
By looking into pmm2-server-autobuild Jenkins logs :)